.github/issue_template.md

```diff
@@ -0,0 +1,40 @@
+Note: This is just a template, so feel free to use/remove the unnecessary things
+
+### Description
+- Type: Bug | Enhancement\Feature Request | Question
+- Priority: Blocker | Major | Minor
+
+---------------------------------------------------------------
+## Bug
+
+**OS**  
+linux|windows|??
```

Comment:
We should have mbed OS as the *first* option here. We should also list baremetal, and instead of '??' list other.